core/dune

```diff
@@ -13,13 +13,9 @@
   (modes native)
   (libraries str yojson ppx_deriving_yojson.runtime unix safepass base64
              ANSITerminal linenoise cohttp lwt websocket websocket-lwt.cohttp
-             findlib menhirLib)
+             findlib menhirLib links_lens)
```

Comment:
I suppose this should be `links.lens` rather than `links_lens`, as the former denotes a subpackage, unless I am mistaken.

Reply:
Changed this.